src/cli_util.rs

```diff
@@ -2028,6 +2028,35 @@ impl ValueParserFactory for RevisionArg {
     }
 }
 
+fn resolve_default_command(config: &config::Config, app: &Command, string_args: &mut Vec<String>) {
+    const PRIORITY_FLAGS: &[&str] = &["help", "--help", "-h", "--version", "-V"];
+
+    let has_priority_flag = string_args
+        .iter()
+        .any(|arg| PRIORITY_FLAGS.contains(&arg.as_str()));
+    if has_priority_flag {
+        return;
+    }
+
+    let app_clone = app
+        .clone()
+        .allow_external_subcommands(true)
+        .ignore_errors(true);
```

Comment:
Nice! I was hoping it could be done something like this but I didn't think of `ignore_errors()` when I looked into it yesterday. (No action needed.)